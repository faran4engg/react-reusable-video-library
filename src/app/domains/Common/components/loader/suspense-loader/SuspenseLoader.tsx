import { FC } from 'react';
import './style.css';

const SuspenseLoader: FC = () => (
  <div id="preloader">
    <div id="loader" />
  </div>
);

export default SuspenseLoader;
