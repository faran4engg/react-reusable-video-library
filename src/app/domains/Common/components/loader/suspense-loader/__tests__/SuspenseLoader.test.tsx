import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import SuspenseLoader from '../SuspenseLoader';

const renderer = createRenderer();

describe('<SuspenseLoader />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<SuspenseLoader />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
