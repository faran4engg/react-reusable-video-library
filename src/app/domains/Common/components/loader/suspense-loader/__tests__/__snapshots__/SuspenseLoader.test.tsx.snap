// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuspenseLoader /> should render and match the snapshot 1`] = `
<div
  id="preloader"
>
  <div
    id="loader"
  />
</div>
`;
